dts/msm8916/msm8916-samsung-gprime.dtsi

```diff
@@ -0,0 +1,18 @@
+lk2nd,keys = <KEY_HOME 109 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+
+samsung,muic-reset {
+	i2c-gpio-pins = <2 3>;
+	i2c-address = <0x25>;
+};
+
+panel {
+	compatible = "samsung,gprime-panel";
+
+	ss_dsi_panel_HX8389C_GH9607501A_QHD {
+		compatible = "samsung,hx8389c-gh9607501a";
+	};
+
+	ss_dsi_panel_S6D78A0_GH9607501A_QHD {
+		compatible = "samsung,s6d78a0-gh9607501a";
+	};
```

Comment:
There is only one panel `HX8389C` (not `HX8389C_GH9607501A`) for fortunaltezt.

Reply:
Can you please suggest the necessary change ?

Reply:
Leaving it as-is doesn't seem to be a problem since https://github.com/gprime-mainline/linux/commit/e6541148af616515a52e5d4fefef61ead651cff8 overrides `compatible = "samsung,gprime-panel";`. This could be a minor coding style issue. I have some ideas:
1. Leave it
2. https://github.com/gprime-mainline/lk2nd/commit/68f1465706ea9a70bbd2ec72dc6262dea0cac60b
3. Don't have `msm8916-samsung-gprime.dtsi` included on fortunaltezt